docs/conf.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8 -*-
+#
+# Elodie documentation build configuration file, created by
+# sphinx-quickstart on Fri Jan  8 14:42:49 2016.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import os
+import shlex
+import sys
+
+import mock
+
+# Add the parent folder to the Python path so Sphinx can import elodie modules.
+sys.path.insert(0, os.path.abspath('..'))
+
+# Mock out the pyexiv2 module so we don't have to install it when we build
+# docs on ReadTheDocs.
+sys.modules['pyexiv2'] = mock.Mock()
```

Comment:
Mocking pyexiv2 since RTD can't install it (see their [FAQ on this](http://read-the-docs.readthedocs.org/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules)).